-1

Tested at: c0a38e187b4a9bf4debaff47464813aad2986994

CMSSW: CMSSW_11_1_X_2020-05-01-1100
SCRAM_ARCH: slc7_amd64_gcc820
You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-5a0cc5/5949/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found compilation error when building: 

<pre>>> Compiling edm plugin /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_1_X_2020-05-01-1100/src/SimCalorimetry/EcalZeroSuppressionProducers/src/EcalZeroSuppressionProducer.cc 
>> Compiling edm plugin /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_1_X_2020-05-01-1100/src/SimCalorimetry/EcalZeroSuppressionProducers/src/ESZeroSuppressionProducer.cc 
>> Compiling edm plugin /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_1_X_2020-05-01-1100/src/SimCalorimetry/EcalZeroSuppressionProducers/src/SealModule.cc 
In file included from /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_1_X_2020-05-01-1100/src/SimCalorimetry/EcalZeroSuppressionProducers/interface/EcalZeroSuppressionProducer.h:4,
                 from /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_1_X_2020-05-01-1100/src/SimCalorimetry/EcalZeroSuppressionProducers/src/EcalZeroSuppressionProducer.cc:2:
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_1_X_2020-05-01-1100/src/CalibCalorimetry/EcalTrivialCondModules/interface/EcalTrivialConditionRetriever.h:297:43: error: 'EcalDataFrame' was not declared in this scope
   std::vector<ROOT::Math::SVector<double, EcalDataFrame::MAXSAMPLES> >
                                           ^~~~~~~~~~~~~
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_1_X_2020-05-01-1100/src/CalibCalorimetry/EcalTrivialCondModules/interface/EcalTrivialConditionRetriever.h:297:43: note: suggested alternative: 'EcalFunParams'
   std::vector<ROOT::Math::SVector<double, EcalDataFrame::MAXSAMPLES> >
                                           ^~~~~~~~~~~~~
</pre>

--- a/CalibCalorimetry/EcalTrivialCondModules/BuildFile.xml
+++ b/CalibCalorimetry/EcalTrivialCondModules/BuildFile.xml
@@ -10,4 +10,5 @@
 <use   name="CondFormats/ESObjects"/>
 <use   name="CondFormats/DataRecord"/>
 <use   name="DataFormats/EcalDetId"/>
+<use   name="DataFormats/EcalDigi"/>
 <use   name="clhep"/>

--- a/CalibCalorimetry/EcalTrivialCondModules/interface/EcalTrivialConditionRetriever.h
+++ b/CalibCalorimetry/EcalTrivialCondModules/interface/EcalTrivialConditionRetriever.h
@@ -113,6 +113,7 @@
 #include "SimG4CMS/Calo/interface/EnergyResolutionVsLumi.h"
 #include "SimG4CMS/Calo/interface/EvolutionECAL.h"
 
+#include "DataFormats/EcalDigi/interface/EcalDataFrame.h"
 // forward declarations
 
 namespace edm {

